sources/packages/api/src-sql/Reports/Create-reports-config.sql

```diff
@@ -0,0 +1,32 @@
+--Create table sims.reports_config
+CREATE TABLE IF NOT EXISTS sims.reports_config(
+    id SERIAL PRIMARY KEY,
+    report_name VARCHAR(100) UNIQUE NOT NULL,
+    report_sql TEXT NOT NULL,
+    -- Audit columns
+    created_at timestamp with time zone NOT NULL DEFAULT now(),
+    updated_at timestamp with time zone NOT NULL DEFAULT now(),
+    creator INT NULL DEFAULT NULL REFERENCES users(id) ON DELETE
```

Comment:
pls add sims.users(id)

Reply:
Are we having issues on migration:revert?

Reply:
yup... if you use `users(id)`. irrespective of `sims.` in migrations scripts, need to always test `migration:revert` for all migrations

Reply:
Tried the same through make make local-api-detached make api and npx typeorm migration:revert

The migration revert worked.

![image](https://user-images.githubusercontent.com/54600590/170163244-23322910-6146-4c5a-bca4-097424bd4206.png)